.devcontainer/amd64/devcontainer.json

```diff
@@ -5,7 +5,9 @@
 
   "runArgs": [
     "--network=host",
-    "--volume=/dev:/dev",
+    "--privileged",
+    "--volume=/dev/v4l/by-id:/dev/v4l/by-id",
+    "--volume=/dev/serial/by-id:/dev/serial/by-id",
```

Comment:
Interesting, so --privileged gives the container access to the host filesystem so that it can lookup the symlinks in /dev/v4l/by-id?